<launch>

    <arg name="UAV_NAME" default="$(optenv UAV_NAME uav15)"/>

    <group ns="$(arg UAV_NAME)">
        <node pkg="mrs_openswarm_utils" type="save_position_node.py" name="save_position_node" output="screen" respawn="false">
            <remap from="~odom" to="estimation_manager/odom_main" />
        </node>
    </group>

</launch>